modules/database/variables.tf

```diff
@@ -36,6 +36,18 @@ variable "database_engine" {
   description = "Which database engine to use, currently supports `postgres` or `mysql`"
 }
 
+variable "database_engine_version" {
+  description = "Which database version to use. See the aws cli describe-db-engine-versions command for a list of valid versions"
```

Comment:
```suggestion
  description = "Which database version to use. See the aws rds describe-db-engine-versions command for a list of valid versions"
```

Reply:
should these be `aws rds describe-db-engine-versions` instead?

Reply:
I meant to check the command in the aws cli, but yeah that works too.